Block/Js.php

```diff
@@ -172,6 +172,10 @@ private function getModuleVersion()
      */
     private function getSentryDSN()
```

Comment:
Why does this function still exist if it always returns ""?  Is there a plan to update the url?